Open Source/Git1/Ashutosh-Khanduala.txt

```diff
@@ -1,4 +0,0 @@
-Ashutosh Khanduala
-ETC A 2nd Yr
-1902070030
-I love Open source
```

Comment:
Why did you delete this file?

Reply:
Didn't know had to keep the original one.

Reply:
it is clearly told in task to "create a new file"

So dont worry...  create a new file and restore Ashutosh-khanduala.txt

Reply:
push again

This PR will be updated automatically after you push it

Reply:
Done